g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java

```diff
@@ -117,7 +118,7 @@ public void addInterceptors(InterceptorRegistry registry) {
 		 */
 
 		// Request Validation
-		InterceptorRegistration apival = registry.addInterceptor(new APIValidationInterceptor(bundleFileBasePath)).addPathPatterns("/**").excludePathPatterns(API.I18N_API_ROOT+"doc/**");
+		InterceptorRegistration apival = registry.addInterceptor(new APIValidationInterceptor(productService.getWhiteList())).addPathPatterns("/**").excludePathPatterns(API.I18N_API_ROOT+"doc/**");
```

Comment:
Can this implementation apply to both saas and on-prem scenario?

Reply:
yes